langs/base.go

```diff
@@ -21,6 +21,7 @@ func init() {
 	registerHelper(&RubyLangHelper{})
 	registerHelper(&RustLangHelper{})
 	registerHelper(&KotlinLangHelper{})
+	registerHelper(&ClojureLangHelper{version: "1.9.0"})
```

Comment:
overall, looks good, need to do few manual tests.

in a meantime, can you please add tests for this helper?
take a look at:
 - https://github.com/fnproject/cli/blob/master/test/cli_lang_boilerplate_test.go#L15

Reply:
Sure, will do.